### ISSUE TITLE
RemoteSSH Icon has changed color.

### ISSUE THREAD

**Issue by devomaa** [dunstontc/dark-plus-syntax#11]
**Video**:
[![IMAGE ALT TEXT HERE](https://img.youtube.com/vi/TwrcCTGKEVU/0.jpg)](https://www.youtube.com/watch?v=TwrcCTGKEVU)

**Comment by szymonos** [comment]
Yes, please fix this. It's hard to recognize if you're working on a remote without it.

**Comment by tcd** [comment]
Currently looking into this.
